go.mod

```diff
@@ -14,7 +14,6 @@ require (
 	github.com/sergi/go-diff v1.0.0
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
-	golang.org/x/crypto v0.0.0-20190308221718-c2843e01d9a2 // indirect
```

Comment:
I had to do this to make make-verify happy since golang got updated to 1.13 in prow. For more context see [Slack](https://kubernetes.slack.com/archives/C8TSNPY4T/p1573086005417200).

Reply:
We rolled back to 1.12 in https://github.com/kubernetes/test-infra/pull/15166, can you revert this change?

Reply:
done